Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -2584,7 +2584,7 @@ class CommandResults:
     """
     def __init__(self, outputs_prefix=None, outputs_key_field=None, outputs=None, indicators=None, readable_output=None,
                  raw_response=None):
-        # type: (str, str, object, list, str, object) -> None
+        # type: (str or None, str or None, object or None, list or None, str or None, object or None) -> None
```

Comment:
thanks for the contribution !

better to use `Optional` instead of `or None`, so for example you can use `Optional[str]`

please add release notes as well